service/datasets/example.csv

```diff
@@ -1,2 +1,1001 @@
 a,b,c
```

Comment:
Edi just mentioned to me that it's better not to upload data to the GitHub, so you might want to delete this file from the remote.